spec/controllers/protected_resources_controller_spec.rb

```diff
@@ -174,4 +174,101 @@ def doorkeeper_unauthorized_render_options(error: nil)
       end
     end
   end
+
+  context 'when custom forbidden render options are configured' do
+    after do
+      module ControllerActions
+        def doorkeeper_forbidden_render_options(error: nil)
+        end
+      end
+    end
+
+    controller do
+      before_filter -> { doorkeeper_authorize! :write }
+
+      include ControllerActions
+    end
+
+    let(:token_string) { '1A2DUWE' }
+
+    context 'with a JSON custom render' do
+      before do
+        module ControllerActions
+          def doorkeeper_forbidden_render_options(error: nil)
+            { json: { error_message: 'Forbidden' } }
+          end
+        end
+      end
+
+      it 'renders a custom JSON response' do
+        forbidden_render_test
+        expect(response.content_type).to eq('application/json')
+        expect(response.status).to eq 403
+        parsed_body = JSON.parse(response.body)
+        expect(parsed_body).not_to be_nil
+        expect(parsed_body['error_message']).to match('Forbidden')
+      end
+    end
+
+    context 'with a status and JSON custom render' do
+      before do
+        module ControllerActions
+          def doorkeeper_forbidden_render_options(error: nil)
```

Comment:
Unused method argument - `error`. You can also write as `doorkeeper_forbidden_render_options(*)` if you want the method to accept any arguments but don't care about them.